appj.adoc

```diff
@@ -0,0 +1,409 @@
+
+When an interpolation method is referred to as linear or quadratic, it means that the method is linear or quadratic in the indices of the interpolation dimensions. 
+
+For convenience, an interpolation variable `s` is introduced, calculated as a function of the index in the target domain of the coordinate value to be reconstituted. In the case of one dimensional interpolation the interpolation variable is computed as 
+
+`s = s(ia, ib, i) = (i - ia)/(ib - ia)`
+
+where `ia` and `ib` are the indices in the target domain of the tie points `A` and `B` and `i` is the index in the target domain of the coordinate value to be reconstituted. 
+
+Note that the value of `s` varies from `0.0` at the tie point `A` to `1.0` at tie point `B`. For example, if `ia = 100` and `ib = 110` and the index in the target domain of the coordinate value to be reconstituted is `i = 105`, then `s = (105 - 100)/(110 - 100) = 0.5`.
+
+In the case of two dimensional interpolation, the two interpolation variables are equivalently computed as
+
+`s1 = s(ia1, ib1, i1) = (i1 - ia1)/(ib1 - ia1)`  +
+`s2 = s(ia2, ic2, i2) = (i2 - ia2)/(ic2 - ia2)`
+
+where `ia1` and `ib1` are the first dimension indices in the target domain of the tie points `A` and `B` respectively, `ia2` and `ic2` are the second dimension indices in the target domain of the tie points `A` and `D` respectively and the indices `i1` and `i2` are the first and second dimension indices respectively in the target domain of the coordinate value to be reconstituted. 
+
+For the reconstitution of the uncompressed coordinate and auxiliary coordinate variables the interpolation method can be applied independently for each interpolation zone, making it possible to parallelize the computational process.
+
+The following notation is used: +  
+A variable staring with `v` denotes a vector and `v.x` , `v.y` and `v.z` refer to the three coordinates of that vector. +
+A variable staring with `ll` denotes a latitude-longitude coordinate pair and `ll.lat` and `ll.lon` refer to the latitude and longitude coordinates. +
+For one dimensional interpolation, `i` is an index in the interpolation dimension, `tp` is an index in the tie point interpolation dimension and `iz` is an index in the interpolation zone dimensions. 
+For two dimensional interpolation, `i1` and `i2` are indices in the interpolation dimensions, `tp1` and `tp2` are indices in the tie point interpolation dimensions and `iz1` and `iz2` are indices in the interpolation zone dimensions. +
```

Comment:
Could we call "tp1" and "tp2" `tpi1` and `tpi2` instead? This makes the naming consistent with the indices (`i1`, `iz`) wand avoids confusion (that I have had) that "tp1" is a an index, not a tie point.

Also, `iz1` is an interpolation zone index - should/could that be now `is1` for interpolation subarea index?

This would also affect figure 1.

Reply:
Sure, that would be fine. I will update accordingly.
Anders